src/renderer/components/widgets/channelSettings/ChannelSettingsModal.tsx

```diff
@@ -57,25 +57,36 @@ const styles = theme => ({
   content: {
     marginLeft: 32
   }
-})
+}))
 
 const handleChange = (setCurrentTab, clearCurrentOpenTab, value) => {
   clearCurrentOpenTab()
   setCurrentTab(value)
 }
 
-export const ChannelSettingsModal = ({
-  classes,
+interface ChannelSettingsModalProps {
+  open: boolean
+  handleClose: () => void
+  currentTab: string
+  channel: Contact
+  isOwner: boolean
+  modalTabToOpen: () => string
+  setCurrentTab: () => void
+  clearCurrentOpenTab: () => void
+}
+
+export const ChannelSettingsModal: React.FC<ChannelSettingsModalProps> = ({
   open,
   handleClose,
   currentTab,
-  channel,
+  channel = {},
```

Comment:
hmmm... `channel` is declared as `Contact`, but here you pass `{}` and it doesn't complain.
Does `Contact` only have optional parameters? How does this work ?